notebooks/250-music-generation/250-music-generation.ipynb

```diff
@@ -0,0 +1,742 @@
+{
```

Comment:
<p>We need inference without gradio to check on CI if the model works.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1242/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='36'/>

Reply:
<p>added additional cell</p>